types/bitcoinjs-lib/index.d.ts

```diff
@@ -232,172 +352,969 @@ export const opcodes: {
+            };
+
+        }
+
+        namespace swap32 {
+            const prototype: {
+            };
+
+        }
+
+        namespace swap64 {
+            const prototype: {
+            };
+
+        }
+
+        namespace toJSON {
+            const prototype: {
+            };
+
+        }
+
+        namespace toLocaleString {
+            const prototype: {
+            };
+
+        }
+
+        namespace toString {
+            const prototype: {
+            };
+
+        }
+
+        namespace ucs2Slice {
+            const prototype: {
+            };
+
+        }
+
+        namespace ucs2Write {
+            const prototype: {
+            };
+
+        }
+
+        namespace utf8Slice {
+            const prototype: {
+            };
+
+        }
+
+        namespace utf8Write {
+            const prototype: {
+            };
+
+        }
+
+        namespace write {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeDoubleBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeDoubleLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeFloatBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeFloatLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt16BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt16LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt32BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt32LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt8 {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeIntBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeIntLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt16BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt16LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt32BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt32LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt8 {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUIntBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUIntLE {
+            const prototype: {
+            };
+
+        }
+
+    }
+
+
+}
+
+
+
+export namespace address {
+    function fromBase58Check(address: string): string;
+
+    function fromOutputScript(outputScript: Buffer, network: Network): string;
+
+    function toBase58Check(hash: Buffer, version: number, ...args: any[]): any;
```

Comment:
Remove the parameter `...args` as it is unused (exists only because `arguments` is used in function for type checking).
Returns `string`.